core/src/main/java/com/distkv/core/KVStore.java

```diff
@@ -1,18 +1,18 @@
 package com.distkv.core;
 
+import com.distkv.core.concepts.DistkvBaseOperation;
+import com.distkv.core.concepts.DistkvDicts;
 import com.distkv.core.concepts.DistkvInts;
+import com.distkv.core.concepts.DistkvLists;
 import com.distkv.core.concepts.DistkvSets;
 import com.distkv.core.concepts.DistkvSortedLists;
-import com.distkv.core.concepts.DistkvStrings;
-import com.distkv.core.concepts.DistkvLists;
-import com.distkv.core.concepts.DistkvDicts;
 
 /**
  * The api of distkv store.
  */
 public interface KVStore {
 
-  DistkvStrings strs();
+  DistkvBaseOperation strs();
```

Comment:
Why `strs()` returns `DistkvBaseOperation` while `lists()` returning `DistkvLists`?

Reply:
i will fix this.